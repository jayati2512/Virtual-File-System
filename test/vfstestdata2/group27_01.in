createvfs MyVFS/@1
createvfs @MyVFS01 100
mountvfs @MyVFS01
makedir /Dir1/ SubD!r1
movedir /Dir1/SubD!r1/ /Dir2/
mountvfs @MyVFS01
unmountvfs @MyVFS01
addfile /Dir1/SubD!r1/ m.em/ory.txt /usr/include/memory.h
mountvfs @MyVFS01
addfile /Dir1/SubD!r1/ poll.txt /usr/include/poll.h
updatefile /Dir1/SubD!r1/polling.txt /usr/include/memory.h
unmountvfs @MyVFS01
updatefile /Dir1/SubD!r1/poll.txt /usr/include/ary.h
