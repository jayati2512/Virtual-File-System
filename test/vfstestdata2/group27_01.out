createvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
movedir_FAILURE CANNOT_FIND_SPECIFIED_DESTINATIONDIR
mountvfs_FAILURE VFS_ALREADY_MOUNTED
unmountvfs_SUCCESS
addfile_FAILURE VFS_NOT_MOUNTED
mountvfs_SUCCESS
addfile_SUCCESS
updatefile_FAILURE INTERNAL_FILE_NOT_FOUND
unmountvfs_SUCCESS
updatefile_FAILURE VFS_NOT_MOUNTED
